.github/workflows/pr-realm-js.yml

```diff
@@ -92,15 +93,15 @@ jobs:
 
       - name: Restore NPM cache
         id: npm-cache
-        uses: actions/cache@v3
+        uses: actions/cache@v4
         with:
           path: ${{ steps.npm-cache-dir.outputs.dir }}
           key: ${{ runner.os }}-node-${{ hashFiles('package-lock.json') }}
           restore-keys: |
             ${{ runner.os }}-node-
 
       - name: Setup Wireit cache
-        uses: google/wireit@setup-github-actions-caching/v1
+        uses: takameyer/wireit@wireit-caching-gha
```

Comment:
I don't think this is strictly needed here, I would rather like that we make them upgrade their action and upgrade when its available instead of forking. For that I've created an issue on their repo: https://github.com/google/wireit/issues/1039

Reply:
I'm fine with that.  Thanks for creating the issue.  Was planning on creating a PR, but wasn't sure how to make a PR against a tag :shrug: